server/src/main/java/org/elasticsearch/cluster/metadata/SingleNodeShutdownMetadata.java

```diff
@@ -139,6 +164,11 @@ public SingleNodeShutdownMetadata(StreamInput in) throws IOException {
         } else {
             this.targetNodeName = null;
         }
+        if (in.getTransportVersion().onOrAfter(SIGTERM_ADDED_VERSION)) {
```

Comment:
This needs to be a new TransportVersion. These cannot be reused.

Reply:
Yup, that code was drafted before the `TransportVersion` change over, it will change before marking for review.